source/macos/AdaptiveCards/AdaptiveCardsTests/Renderers/ColumnRendererTests.swift

```diff
@@ -53,13 +53,14 @@ class ColumnRendererTests: XCTestCase {
         var columnView = renderColumnView()
         XCTAssertEqual(columnView.arrangedSubviews.count, 1)
         
+        // since we removed padding view all together and replaced with lastPadding but use paddingView for vertical content alignment this has changed
         column = .make(items: [FakeTextBlock.make()], verticalContentAlignment: .center)
         columnView = renderColumnView()
-        XCTAssertEqual(columnView.arrangedSubviews.count, 3)
+        XCTAssertEqual(columnView.arrangedSubviews.count, 4)
         
         column = .make(items: [FakeTextBlock.make()], verticalContentAlignment: .bottom)
         columnView = renderColumnView()
-        XCTAssertEqual(columnView.arrangedSubviews.count, 2)
+        XCTAssertEqual(columnView.arrangedSubviews.count, 3)
```

Comment:
Why count is increase? if content at center then there is already one _**stretchableView**_ is present, if i'm not wrong then why subviews count +1.